functests/functests.go

```diff
@@ -0,0 +1 @@
+package functests
```

Comment:
Do you need this file?

Reply:
Nopes removed t.

Reply:
Refer necessary changes in this commit c6c6970